libr/include/r_util/r_str.h

```diff
@@ -184,7 +184,6 @@ R_API const char *r_sub_str_rchr(const char *str, int start, int end, char chr);
 R_API char *r_str_ichr(char *str, char chr);
 R_API bool r_str_ccmp(const char *dst, const char *orig, int ch);
 R_API bool r_str_cmp_list(const char *list, const char *item, char sep);
-R_API R_DEPRECATE int r_str_cmp(const char *dst, const char *orig, int len);
```

Comment:
we cant break the ABI, so instead of removing this line just put a comment like `// R2_590`, so it will be removed when the day arrives

Reply:
fixed by putting comment

// R2_590 R_DEPRECATE